### DISCUSSION TITLE
Why a user can use localdomain eamil address as register email?

### DISCUSSION THREAD

**Discussion by X-21** [community/community#22767] [New to GitHub]
<p><a href="https://github.community/t/why-a-weird-email-linked-to-a-wrong-user/136334/2">https://github.community/t/why-a-weird-email-linked-to-a-wrong-user/136334/2?u=x-21</a></p>
<p>It looks like this user use an illegal email address and verify it. How did he make it?</p>

**Comment by canuckjacq** [comment]
<p>Unverified email addresses can be used for commit attribution. I hope that clears things up!</p>

**Comment by X-21** [comment]
<p>But why this commit was linked to the user?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>If you add an email to your account, and commit using that email, it will be attributed to your account. It doesn’t need to be verified.</p>

**Comment by X-21** [comment]
<p>I got it, thank you canuckjacq.</p>
